app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -0,0 +1,31 @@
+package org.simple.clinic.bp.history
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import org.simple.clinic.bp.BloodPressureMeasurement
+import org.simple.clinic.main.TheActivity
+import java.util.UUID
+
+class BloodPressureHistoryScreen(
+    context: Context,
+    attrs: AttributeSet
+) : ConstraintLayout(context, attrs), BloodPressureHistoryScreenUi, BloodPressureHistoryScreenUiActions {
+
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+    if (isInEditMode) {
+      return
+    }
+    TheActivity.component.inject(this)
```

Comment:
We should use the newer way to inject screens instead of the older way.

```kotlin
context.injector<BloodPressureHistoryScreenInjector>().inject(this)
```

Reply:
Didn't knew we switched to an extension. Fixed it 👍